prow/config/plugins.yaml

```diff
@@ -41,6 +43,17 @@ require_matching_label:
       Whether it refers to an issue or to a PR in [Falco](https://github.com/falcosecurity/falco).
       Please specify it either using `/kind <group>` or manually from the side menu.
       In case you do not know which kind this proposal is please mention the maintainers using `@team/maintainers`.
+  - missing_label: needs-kind
+    org: falcosecurity
+    repo: falco
+    issues: true
+    regexp: ^kind/
+  - missing_label: needs-kind
+    org: falcosecurity
+    repo: falco
+    prs: true
+    regexp: ^kind/
```

Comment:
```suggestion
    regexp: ^kind/
     missing_comment: |
      There is not a label identifying the kind of this PR.
      Please specify it either using `/kind <group>` or manually from the side menu.
      In case you do not know which kind this proposal is please mention the maintainers using `@team/maintainers`.
--



```